README.md

```diff
@@ -40,6 +40,23 @@ Optionally you can pass a list of services to be rebuilt:
 ./bootstrap.sh up api web
 ```
 
+## Services pre-run setup
```

Comment:
The description of the repository is quite long already. With this point, it will get even longer. Maybe it's time to add some table of contents? We can also use `<details>` tag for more extended code examples to make it even easier to navigate through it. https://gist.github.com/ericclemmons/b146fe5da72ca1f706b2ef72a20ac39d for

Reply:
I'll leave this for another PR.